debian/control

```diff
@@ -1,1198 +0,0 @@
-Source: ceph
```

Comment:
why do we need to remove `debian/control` ? how does this help with crimson-osd's mkfs support?